avro-converter/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -1500,6 +1500,48 @@ public void testCacheSchemaToConnectConversion() {
     assertEquals(2, cache.size());
   }
 
+  @Test
+  public void testAvroWithAndWithoutMetaData() {
+    String s1 = "{\n"
```

Comment:
I don't find these json records encoded as java strings easy to read nor easy to edit. 
Should we use jackson and create the json object programmatically as we do elsewhere in this test?